corehq/apps/commtrack/forms.py

```diff
@@ -76,15 +80,15 @@ def __init__(self, *args, **kwargs):
             ),
             Fieldset(
                 _('Consumption Settings'),
-                'use_auto_consumption',
+                PrependedText('use_auto_consumption', ''),
                 'consumption_min_transactions',
                 'consumption_min_window',
                 'consumption_optimal_window',
-                'individual_consumption_defaults',
+                PrependedText('individual_consumption_defaults', ''),
             ),
             Fieldset(
                 _('Phone Settings'),
-                'sync_consumption_fixtures',
+                PrependedText('sync_consumption_fixtures', ''),
             ),
             ButtonHolder(
```

Comment:
see the `FormActions` part in https://www.commcarehq.org/styleguide/forms/#b3-migration